### PR TITLE
Implementing the new methods added to the AuthorizationContext

### PR DESCRIPTION
**Pull request by Nishith**

### CODE REVIEW

File: cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java [hunk 1]
```diff
@@ -100,6 +102,32 @@ public Properties getExtensionProperties() {
       }
 
       @Override
+      public void createTopic(String s) throws TopicAlreadyExistsException, IOException {
```
Comment (anwar6953) [inline cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java:105]: Just a thought... isn't it weird to have such methods in a `AuthorizationContext` class?

**Comment by anwar6953** [comment]
LGTM to unblock compilation.

File: cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java (hunk 1 above)
Comment (Nishith) [reply cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java:105]: It is, but that is a TMS question.

Comment (anwar6953) [reply cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java:105]: For now, filed: https://issues.cask.co/browse/CDAP-7994

Comment (chtyim) [reply cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java:105]: It is not a TMS question. It is the authorization API issue. See my comment in CDAP-7994.

**Comment by chtyim** [comment]
BTW, adding new method to api that requires changing code in this repo also reflect the issue in the auth API design. At the very least, it should provide test/mock classes, instead of requiring the extension implementation comes up with its own one.

**Pull request merged**